release.md

```diff
@@ -34,6 +65,10 @@ in a release branch with a version tag (vX.X.X).  Then, master will be updated
 to the next minor version.  This can always be found in the
 [version file](cirq/_version.py).
 
+### Release Schedule
+Releases are made on an as-needed basis determined by Cirq maintainers. All Cirq
+packages (including vendor packages such as `cirq-aqt`) are released at the same
+time.
```

Comment:
I'm ok with this approach, though it is unclear how things would unfold should a vendor package require an urgent patch because it seems that `cirq-core` dictates the release schedule. I agree it should be a rare event, but I would still like to see it documented in the "one-off process" you mentioned above.

Reply:
Agree that this process needs to be documented. I have a proposal (in short, making a minor version release branch when a patch is necessary) but it still needs to be discussed with a wider audience. Because this is rare, the plan is to defer the discussion to unblock releasing the version policy.